include/core/parameters_lagrangian.h

```diff
@@ -201,6 +200,8 @@ namespace Parameters
       std::vector<double> list_x, list_y, list_z, list_vx, list_vy, list_vz,
         list_wx, list_wy, list_wz, list_d;
 
+      std::string insertion_particles_file;
```

Comment:
It's more a filename than a file, but it's up to you

Reply:
that is true